corehq/apps/enterprise/static/enterprise/js/project_dashboard.js

```diff
@@ -270,13 +270,13 @@ hqDefine("enterprise/js/project_dashboard", [
         );
 
         kissmetrics.track.event(`[${metricType}] Visited page`);
-        $(".report-panel").each(function () {
+        $(".js-report-panel").each(function () {
```

Comment:
@biyeun I did a brief search, but can you recommend any reading on this practice?